src/lock/session.go

```diff
@@ -121,10 +134,10 @@ func (r *Runner) Run() error {
 }
 
 func (r *Runner) startSession(ctx context.Context, db Database) (sessionID int64, err error) {
-	span, spanCtx := r.tracer.StartSpanWithContext(ctx, "runner start session")
+	span, spanCtx := r.client.StartSpanWithContext(ctx, "runner start session")
```

Comment:
Can you set other tags on this like session_id and so we can capture those in the spans as well